modules/030-cloud-provider-yandex/hooks/yandex_cluster_configuration_test.go

```diff
@@ -98,6 +98,7 @@ sshPublicKey: ssh-rsa test
 withNATInstance:
   internalSubnetID: test
   natInstanceExternalAddress: 84.201.160.148
+  exporterAPIKey: "Off"
```

Comment:
You test here empty exporterAPIKey or value?

Reply:
empty, fixed. thank you